src/averageAges.js

```diff
@@ -20,8 +20,19 @@ function calculateMenAverageAge(people, century) {
   // avoid using loop and forEach
   // replace `if ()` statement with &&, || or ?:
   // without nesting
-}
 
+  const menDiedInThisCentury = people.filter(person => {
+    return century
+      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
+      : person.sex === 'm';
+  });
```

Comment:
```suggestion
  const menDiedInThisCentury = people.filter(person => century
      ? Math.ceil(person.died / 100) === century && person.sex === 'm'
      : person.sex === 'm';
  );
```
it can be simplified